tools/c7n_gcp/c7n_gcp/resources/logging.py

```diff
@@ -18,20 +18,134 @@
 # how to map them given a project level root entity sans use of c7n-org
 
 
-@resources.register('logsink')
+@resources.register('log-sink')
```

Comment:
renaming resources isn't something we do as a rule if its been in a release as it breaks every extant policy.

Reply:
reverted